File: src/test/samples/validSamples/validSample.r
#region FirstRegion
x <- 42
#endregion

# region Second Region
my_function <- function() {
  #region      InnerRegion
  print("Hello")
  #endregion      ends InnerRegion

 #   region
 print("Unnamed region")
    #endregion
}
# endregion

File: src/test/samples/validSamples/validSample.fs
//#region   FirstRegion
let x = 42
//#endregion

// #region Second Region
type MyClass() =
    // #region InnerRegion
    member this.MyMethod() = ()
    // #endregion ends InnerRegion

    // #region
    member this.MyOtherMethod() = ()
    //        #endregion
// #endregion

